src/hotspot/share/utilities/elfFile.cpp

```diff
@@ -789,7 +789,7 @@ bool DwarfFile::DebugAranges::read_set_header(DebugArangesSetHeader& header) {
 
   // We must align to twice the address size.
   uint8_t alignment = DwarfFile::ADDRESS_SIZE * 2;
-  uint8_t padding = alignment - (_reader.get_position() - _section_start_address) % alignment;
+  uint64_t padding = alignment - (_reader.get_position() - _section_start_address) % alignment;
```

Comment:
We're not using 'long' in platform dependent code but this elf support does use it.  That should be changed/fixed together if we decide to change/fix it.  Since this code isn't used on windows, maybe we don't mind the 'long' type here.